WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -2916,10 +2927,14 @@ extension AztecPostViewController {
         restoreInputAssistantItems()
     }
 
-    fileprivate func resetMediaAttachmentOverlay(_ mediaAttachment: MediaAttachment) {
-        mediaAttachment.overlayImage = nil
-        mediaAttachment.message = nil
-        mediaAttachment.shouldHideBorder = false
+    fileprivate func resetMediaAttachmentOverlayIfNoError(_ mediaAttachment: MediaAttachment) {
+        if let uploadID = mediaAttachment.uploadID,
+        let media = self.mediaFor(uploadID: uploadID),
+        media.error == nil {
```

Comment:
⚠️ So, I'm not sure if we need reset only if there's an error. It seems that this is causing a regression. 

1. Edit a post that has images.
2. Select an image. The action sheet should show.
3. Dismiss the action sheet. 

The overlay will not disappear. 

[Here](https://drive.google.com/file/d/1-PiLnH8H-bOf9F49ygA9sRVYkv5dBzat/view?usp=sharing) is a video.

Reply:
> So, I'm not sure if we need reset only if there's an error. It seems that this is causing a regression.

The reset is only if there is *no* error `media.error == nil`
The vidoe doesn't load for some reason. Is it a big file?

Reply:
>  Is it a big file

It's 16 MB. Sorry. I should learn how to optimize videos soon. 😅